internal/jasmine_node_test/jasmine_runner.js

```diff
@@ -8,7 +8,16 @@ try {
   if (global.jasmine) {
     // global.jasmine has been initialized which means a bootstrap script
     // has already required `jasmine-core` and called jasmineCore.boot()
-    jasmineCore = global.jasmineCore || require('jasmine-core');
+    jasmineCore = global.jasmineCore;
+    if (!jasmineCore) {
+      try {
+        // Try unhoisted jasmine-core first so that we don't
+        // need an @npm//jasmine-core dep in the unhoisted case.
+        jasmineCore = require('jasmine/node_modules/jasmine-core');
+      } catch (_) {
```

Comment:
you could check if the error.code is the MODULE_NOT_FOUND one

Reply:
good call thanks